tests/js/spec/views/organizationDiscover/discover.spec.jsx

```diff
@@ -203,26 +208,99 @@ describe('Discover', function() {
         expect(queryBuilder.reset.mock.calls).toHaveLength(prevCallCount);
       });
     });
+  });
 
-    describe('saved query', function() {
-      it('resets saved query', function() {
-        const wrapper = mount(
-          <Discover
-            queryBuilder={queryBuilder}
-            organization={organization}
-            params={{}}
-            savedQuery={TestStubs.DiscoverSavedQuery()}
-          />,
-          TestStubs.routerContext()
-        );
-        wrapper.instance().updateField('fields', ['message']);
-        wrapper.instance().runQuery();
-        wrapper.update();
-        expect(queryBuilder.getInternal().fields).toEqual(['message']);
-        wrapper.instance().reset();
-        wrapper.update();
-        expect(queryBuilder.getInternal().fields).toEqual(['test']);
+  describe('Saved query', function() {
+    let wrapper, deleteMock, updateMock;
+    beforeEach(function() {
+      wrapper = mount(
+        <Discover
+          queryBuilder={queryBuilder}
+          organization={organization}
+          params={{}}
+          savedQuery={TestStubs.DiscoverSavedQuery()}
+          updateSavedQueryData={() => {}}
+          view="saved"
+        />,
+        TestStubs.routerContext()
```

Comment:
what's this do?

Reply:
It mounts the component with this context https://github.com/getsentry/sentry/blob/master/tests/js/setup.js#L164. In this case I think we just need it to make the emotion theme stuff work